pkg/trustedresources/verify.go

```diff
@@ -46,8 +46,18 @@ const (
 // Return an error when no policies are found and trusted-resources-verification-no-match-policy is set to fail,
 // or the resource fails to pass matched enforce verification policy
 // source is from ConfigSource.URI, which will be used to match policy patterns. k8s is used to fetch secret from cluster
-func VerifyTask(ctx context.Context, taskObj v1beta1.TaskObject, k8s kubernetes.Interface, source string, verificationpolicies []*v1alpha1.VerificationPolicy) error {
-	matchedPolicies, err := getMatchedPolicies(taskObj.TaskMetadata().Name, source, verificationpolicies)
+func VerifyTask(ctx context.Context, taskObj *v1beta1.Task, ct *v1beta1.ClusterTask, k8s kubernetes.Interface, source string, verificationpolicies []*v1alpha1.VerificationPolicy) error {
+	// TODO if we can simply convert ClusterTask to Task here for verify?
```

Comment:
also one question it seems that we are also converting ClusterTask (one of the implemented v1beta1.TaskObject) to Task for verification, would it be fine just to convert it to Task here for verify? cc @Yongxuanzhang thanks!

Reply:
yes, clustertask is not included in trusted resources. So it is fine to convert it to task

Reply:
Thanks for confirming!